eng/mgmt/mgmtmetadata/cognitiveservices_resource-manager.txt

```diff
@@ -5,13 +5,18 @@ Generating CSharp code
 Executing AutoRest command
 cmd.exe /c autorest.cmd https://github.com/Azure/azure-rest-api-specs/blob/main/specification/cognitiveservices/resource-manager/readme.md --csharp --version=2.0.4421 --reflect-api-versions --csharp-sdks-folder=C:\Repos\azure-sdk-for-net\sdk
 Autorest CSharp Version: 2.3.82
-2022-06-07 22:30:22 UTC
+2022-09-28 21:44:18 UTC
 Azure-rest-api-specs repository information
 GitHub fork: Azure
 Branch:      main
-Commit:      ad5110c7ba2113d5f77946338231f45ac4d09c82
+Commit:      4b2f5a45a34e351e38e8b075cf1da5a775fd414a
```

Comment:
The mgmt. sdk has moved to Track 2, please just upgrade the https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/cognitiveservices/Azure.ResourceManager.CognitiveServices

Reply:
@ArthurMa1978 maybe I missed, but I don't recall any notification to us about this change.
I don't see any guideline (or checklist) about how to update track 2 SDK neither.

https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#on-boarding-data-plane-gen-1-convenience-clients-and-management-plane-generated-clients

From here (track 1 client SDK guideline) I guess I need to update hash and generate code. but not sure if I need to update history?